packages/fsapp/src/beta-app/router/types.ts

```diff
@@ -175,7 +175,10 @@ export type Route =
  */
 export type Routes = readonly (Route | RouteCollection)[];
 
-export type ExternalRoute = Route & { readonly tabAffinity?: string };
+export type ExternalRoute = Route & {
+  readonly tabAffinity?: string;
+  readonly statusBarStyle?: OptionsStatusBar;
```

Comment:
Is this necessary when `readonly statusBarStyle?: OptionsStatusBar;` exists on `ComponentRoute` which is part of the `Route` union?

I recall you mentioning there was some sort of type error, but what were the conditions of that?
Maybe we can solve that with type guards?